src/farm/library.cairo

```diff
@@ -95,6 +114,14 @@ namespace CarbonableFarmer {
         return ();
     }
 
+    func yielder_initializer{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
```

Comment:
We must keep follow good practices and keep initializer named initializer imo.
Better to create a dedicated namespace for CarbonableYielder with its own initializer I think.